baseline/pytorch/seq2seq/train.py

```diff
@@ -43,7 +44,7 @@ def __init__(self, model, **kwargs):
 
     @staticmethod
     def _num_toks(tgt_lens):
-        return np.sum(tgt_lens)
+        return float(torch.sum(tgt_lens))
```

Comment:
I thought the suggested way to get the value from a pytorch tensor into a python number was to use `.item`?

Also the number of tokens seems like it should be an int not a float?

Reply:
We could.  AFAICT its only ever used to divide out tokens or multiply by another float

Reply:
switched to item